app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):

```

Comment:
![image](https://user-images.githubusercontent.com/103201010/209964963-c9ed575d-85e6-4391-940c-7806110894cf.png)
this class also should inherit from the abc class